tasks/image/steps.go

```diff
@@ -0,0 +1,67 @@
+package image
+
+import (
+	"bytes"
+	"fmt"
+	"html/template"
+	"strings"
+)
+
+type stepInjector string
+
+var customSteps = map[string]string{
+	"GLIDE":  "{{ .GLIDE }}",
+	"GO-GET": "{{ .GOGET }}",
+	"GO-BIN": "{{.GOBIN}}",
+}
```

Comment:
I think it would be good to hold off on adding any custom steps for the first version of this. This can potentially grow quite a bit. We would need some way to allow users to define these without having to add patches to `dobi`.